lib/game.rb

```diff
@@ -1,24 +1,84 @@
+require './lib/ship'
+require './lib/cell'
+require './lib/board'
+require './lib/game'
```

Comment:
We should change this because we don't need to require the file itself on the file we're calling.